core/src/main/java/de/danoeh/antennapod/core/feed/FeedMedia.java

```diff
@@ -186,6 +186,9 @@ public int getPosition() {
 
     public void setPosition(int position) {
         this.position = position;
+        if(position > 0) {
```

Comment:
Perhaps I'm being daft, is this condition backwards?  Why set 'played' to false when position is greater than 0?  Should it be setting 'new' to false?